src/Microsoft.DotNet.Darc/Darc/Operations/UpdateDependenciesOperation.cs

```diff
@@ -120,16 +120,16 @@ public override async Task<int> ExecuteAsync()
                 }
                 catch (RestApiException e) when (e.Response.Status == 404)
                 {
-                    Console.WriteLine($"Could not find build with BAR id '{_options.BARBuildId}'.");
+                    Logger.LogError($"Could not find build with BAR id '{_options.BARBuildId}'.");
                     return Constants.ErrorCode;
                 }
             }
             else if (!_options.CoherencyOnly)
             {
                 if (string.IsNullOrEmpty(_options.Channel))
                 {
-                    Console.WriteLine($"Please supply either a channel name (--channel), a packages folder (--packages-folder) " +
-                                        "a BAR build id (--id), or a specific dependency name and version (--name and --version).");
+                    Logger.LogError($"Please supply either a channel name (--channel), a packages folder (--packages-folder) " +
```

Comment:
```suggestion
                    Logger.LogError("Please supply either a channel name (--channel), a packages folder (--packages-folder) " +
```
looks like we don't need this